### ISSUE TITLE
top by rca api call returns incorrect values

### ISSUE THREAD

**Issue by alexandersimoes** [tesseract-olap/tesseract#161]
Here's my url: https://api.datamexico.org/tesseract/cubes/trade_balance/aggregate.jsonrecords?drilldowns[]=Geography.State&drilldowns[]=Product.HS4&measures[]=Exports&parents=false&sparse=false&rca=Product.HS4,Geography.State,Exports&top=2,Geography.State,rca,desc

and a sample response object:

```js
{
    "State ID": 1,
    "State": "Aguascalientes",
    "HS4 ID": 1,
    "HS4": 64191.99853515625
},
```

But here I would expect the `HS4` column to be the name of the HS4 product and also I think the `HS4 ID` is wrong. Lastly, it is not returning the RCA value itself.

**Comment by hwchen** [comment]
ok, two things:

Perhaps I need to change the behavior, but you can't name the drilldowns in `drilldowns[]` as well as in `rca`. This leads to the situation where you might want to do rca without any drilldowns specified. I'll have to sort this out soon, but I'm wondering if you actually want to do rca by year, which fill the 'necessary drilldown' slot for now, until I allow rca with specified outside drilldowns.

https://api.datamexico.org/tesseract/cubes/trade_balance/aggregate.jsonrecords?drilldowns%5B%5D=Year.Year.Year&measures%5B%5D=Exports&parents=false&sparse=false&rca=Product.HS4,Geography.State,Exports&top=2,Geography.State,rca,desc

**Comment by hwchen** [comment]
closed in favor of #167
